pkg/operator/images.go

```diff
@@ -12,12 +12,5 @@ type Images struct {
 
 // DefaultImages returns default set of images for operator.
 func DefaultImages() Images {
```

Comment:
nit: This is not technically true any longer. I'd say we should either nuke this function all together and move to using `Images{}` *or* rename it to something like `NewImages()` or the like.

Reply:
+1 for using the raw literal